src/js/integrations/debugsymbolicator.ts

```diff
@@ -158,4 +177,34 @@ export class DebugSymbolicator implements Integration {
       event.exception.values[0].stacktrace.frames = frames.reverse();
     }
   }
+
+  /**
+   * This tries to add source context for in_app Frames
+   *
+   * @param frame StackFrame
+   * @param getDevServer function from RN to get DevServer URL
+   */
+  private async _addSourceContext(
+    frame: StackFrame,
+    getDevServer?: any
+  ): Promise<void> {
+    // tslint:disable: no-unsafe-any no-non-null-assertion
+    const response = await fetch(
+      `${getDevServer().url}${frame
+        .filename!.replace(/\/+$/, "")
+        .replace(/.*\//, "")}`,
+      {
+        method: "GET"
+      }
+    );
+
+    const content = await response.text();
+    const lines = content.split("\n");
+
+    const line = Math.max(frame.lineno! - 1, 0);
+    frame.pre_context = lines.slice(line - 6, line);
+    frame.post_context = lines.slice(line, line + 5);
```

Comment:
Maybe I'm reading this wrong, but is there a reason you're grabbing six lines of pre-context but only five lines of post-context? Also, shouldn't the post-context start on the line _after_ the line in question?

Reply:
https://github.com/getsentry/sentry-react-native/pull/741#issuecomment-573024666

Reply:
👍🏻